### PR TITLE
SugarCRM sources 1) Accounts, Contacts 2) Events documentation

### PR DESCRIPTION
**Pull request by mohan-h**
Initial commit

There are 2 SugarCRM sources in this documentation PR commit
1) Accounts, Contacts 
2) Events

### CODE REVIEW

File: help/sources/tutorials/api/create/crm/sugarcrm-accounts-contacts.md [hunk 1]
```diff
@@ -0,0 +1,965 @@
...
+hide: true
+hidefromtoc: true
+---
+# Create a source connection and dataflow for [!DNL SugarCRM Accounts & Contacts] using the Flow Service API
+
+[[!DNL SugarCRM]](https://www.sugarcrm.com/) is a customer relationship management (CRM) system. [!DNL SugarCRM]'s functionality includes sales-force automation, marketing campaigns, customer support, collaboration, Mobile CRM, Social CRM and reporting.
+
+To retrieve relationship data setup within [!DNL SugarCRM], this Adobe Experience Platform [source](https://experienceleague.adobe.com/docs/experience-platform/sources/home.html?lang=en) connector leverages the below [!DNL SugarCRM] APIs :
+   1. [Account API](https://market.apidocs.sugarcrm.com/#b0aeb0cd-80ea-4688-8474-54e4873f32f3) to retrieve your companies, and the 
+   1. [Contact API](https://market.apidocs.sugarcrm.com/#308c5025-9478-4de3-8a41-1fc3cff1d8d1) to retrieve the people.
```
Comment (carlojavier) [inline help/sources/tutorials/api/create/crm/sugarcrm-accounts-contacts.md]: We can shorten this to: The SugarCRM source leverages the [Accounts](https://market.apidocs.sugarcrm.com/#b0aeb0cd-80ea-4688-8474-54e4873f32f3) and [Contacts](https://market.apidocs.sugarcrm.com/#308c5025-9478-4de3-8a41-1fc3cff1d8d1) endpoints of the SugarMarket RESTFUL API to retrieve relationship data setup from SugarCRM.

File: help/sources/tutorials/api/create/crm/sugarcrm-accounts-contacts.md [hunk 2]
```diff
@@ -0,0 +1,965 @@
...
+
+After you bring that data to Experience Platform, you can then execute any analytics.
+
+[!DNL SugarCRM Accounts & Contacts] uses bearer tokens as an authentication mechanism to communicate with the [!DNL SugarCRM] Account and Contact APIs.
+
+## Prerequisites
+
+The first step in creating a [!DNL SugarCRM Accounts & Contacts] source connection is to ensure that you have the below :
+1. A [!DNL SugarMarket] *(SugarCRM’s product for Marketing Automation)* account.
+1. Additionally, a unique API username and user account separate from any user account associated with the marketing or sales process; having API Access permissions in order to access the API. The process to obtain / set up the account is documented on the [[!DNL SugarMarket RESTFUL API (sugarcrm.com)]](https://market.apidocs.sugarcrm.com/#intro) page.
```
Comment (carlojavier) [inline help/sources/tutorials/api/create/crm/sugarcrm-accounts-contacts.md]: Can we add a link to where a user can start creating an account? Suggesting some slight rewording here:

"The first step in creating a SugarCRM Accounts & Contacts source connection is to ensure that you have the following:

* A SugarMarket account. SugarMarket is SugarCRM’s marketing automation platform. To create an account, visit [LINK HERE].
* A unique API username and account separate from any user account associated with the marketing or sales process. This unique username and account combination must have API access permissions. For more information on the process to set up an account, visit the SugarMarket RESTFUL API documentation."

File: help/sources/tutorials/api/create/crm/sugarcrm-accounts-contacts.md [hunk 3]
```diff
@@ -0,0 +1,965 @@
...
+
+### Gather required credentials
+
+In order to connect [!DNL SugarCRM Accounts & Contacts] to Platform, you must provide values for the following connection properties:
+
+| Credential | Description | Example |
+| --- | --- | --- |
+| *`Host`* | API endpoint. | `developer.salesfusion.com` |
```
Comment (carlojavier) [inline help/sources/tutorials/api/create/crm/sugarcrm-accounts-contacts.md]: Can we make the description for Host into full sentence, so: "The API endpoint that the source connects to." or something like that.

File: help/sources/tutorials/api/create/crm/sugarcrm-accounts-contacts.md [hunk 4]
```diff
@@ -0,0 +1,965 @@
...
+| *`Username`* | Your SugarCRM developer account username. | `abc.def@example.com@sugarmarketdemo000.com` |
+| *`Password`* | Your SugarCRM developer account password. | `123456789` |
+
+## Connect [!DNL SugarCRM Accounts & Contacts] to Platform using the [!DNL Flow Service] API
+
+The following tutorial walks you through the steps to create a [!DNL SugarCRM Accounts & Contacts] source connection and create a dataflow to bring [!DNL SugarCRM Accounts & Contacts] data to Platform using the [[!DNL Flow Service] API](https://www.adobe.io/experience-platform-apis/references/flow-service/).
```
Comment (carlojavier) [inline help/sources/tutorials/api/create/crm/sugarcrm-accounts-contacts.md]: we can shorten the middle of this to: "...to create a [!DNL SugarCRM Accounts & Contacts] source connection and dataflow to bring..."

File: help/sources/tutorials/api/create/crm/sugarcrm-accounts-contacts.md [hunk 5]
```diff
@@ -0,0 +1,965 @@
...
+```https
+POST /sourceConnections
+```
+
+**Request**
+
+The following request creates a source connection for [!DNL SugarCRM Accounts & Contacts]:
+
+If you need to work with the [!DNL SugarCRM] Accounts API the `object_type` property value should be `accounts`.
+
```
Comment (carlojavier) [inline help/sources/tutorials/api/create/crm/sugarcrm-accounts-contacts.md:586]: We can circumvent the two examples by using tabs. Essentially, right before the shell example, we can add:

`>[!BEGIN TABS]`

`>[!TAB Accounts]`

*request example here*

`>[!TAB Contacts]`

*request example here*

`>[!ENDTABS]`

Then we can just repeat the same for response. Doc here for additional guidance: https://experienceleague.adobe.com/docs/authoring-guide-exl/using/markdown/syntax-style-guide.html?lang=en#tabs

Comment (carlojavier) [reply help/sources/tutorials/api/create/crm/sugarcrm-accounts-contacts.md:586]: We can apply the same to the explore examples as well

File: help/sources/tutorials/ui/create/crm/sugarcrm-accounts-contacts.md [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+
+After you bring that data to Experience Platform, you can then execute any analytics.
+
+[!DNL SugarCRM Accounts & Contacts] uses bearer tokens as an authentication mechanism to communicate with the [!DNL SugarCRM] Account and Contact APIs.
+
+## Prerequisites
+
+The first step in creating a [!DNL SugarCRM Accounts & Contacts] source connection is to ensure that you have the below :
+1. A [!DNL SugarMarket] *(SugarCRM’s product for Marketing Automation)* account.
+1. Additionally, a unique API username and user account separate from any user account associated with the marketing or sales process; having API Access permissions in order to access the API. The process to obtain / set up the account is documented on the [[!DNL SugarMarket RESTFUL API (sugarcrm.com)]](https://market.apidocs.sugarcrm.com/#intro) page.
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/crm/sugarcrm-accounts-contacts.md]: We can remove this - I will create a separate page to contain the pre-reqs for SugarCRM

File: help/sources/tutorials/api/create/crm/sugarcrm-events.md [hunk 7]
```diff
@@ -0,0 +1,662 @@
...
+
+After you bring that data to Experience Platform, you can then execute any analytics.
+
+[!DNL SugarCRM Events] uses bearer tokens as an authentication mechanism to communicate with the [!DNL SugarCRM] Events API.
+
+## Prerequisites
+
+The first step in creating a [!DNL SugarCRM Events] source connection is to ensure that you have the below :
+1. A [!DNL SugarMarket] *(SugarCRM’s product for Marketing Automation)* account.
+1. Additionally, a unique API username and user account separate from any user account associated with the marketing or sales process; having API Access permissions in order to access the API. The process to obtain / set up the account is documented on the [[!DNL SugarMarket RESTFUL API (sugarcrm.com)]](https://market.apidocs.sugarcrm.com/#intro) page.
```
Comment (carlojavier) [inline help/sources/tutorials/api/create/crm/sugarcrm-events.md]: Let's remove this. I will make a separate page on experience league to contain the pre-reqs for SugarCRM. keep the one for accounts though

File: help/sources/tutorials/ui/create/crm/sugarcrm-events.md [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+
+After you bring that data to Experience Platform, you can then execute any analytics.
+
+[!DNL SugarCRM Events] uses bearer tokens as an authentication mechanism to communicate with the [!DNL SugarCRM] Events API.
+
+## Prerequisites
+
+The first step in creating a [!DNL SugarCRM Events] source connection is to ensure that you have the below :
+1. A [!DNL SugarMarket] *(SugarCRM’s product for Marketing Automation)* account.
+1. Additionally, a unique API username and user account separate from any user account associated with the marketing or sales process; having API Access permissions in order to access the API. The process to obtain / set up the account is documented on the [[!DNL SugarMarket RESTFUL API (sugarcrm.com)]](https://market.apidocs.sugarcrm.com/#intro) page.
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/crm/sugarcrm-events.md]: Same as above, we can remove this

File: help/sources/tutorials/ui/create/crm/sugarcrm-events.md [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+| *`Password`* | Your SugarCRM developer account password. | `123456789` |
+
+### Create Platform Schema
+
+Create a Platform [schema](/help/xdm/schema/composition.md) required for [!DNL SugarCRM Events].
+
+![Platform UI screenshot showing an example schema for SugarCRM Events](../../../../images/tutorials/create/sugarcrm-events/sugarcrm-schema-events.png?lang=en)
+
+>[!WARNING]
+> When mapping the schema ensure you also map the mandatory event_id and timestamp fields required by Platform.
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/crm/sugarcrm-events.md]: Need to add another ">" in between here, and also need to remove the space between ">" and the text, so:
>[!WARNING]
>
>When mapping...

File: help/sources/tutorials/ui/create/crm/sugarcrm-accounts-contacts.md [hunk 10]
```diff
@@ -0,0 +1,109 @@
...
+1. In the Platform UI, select View Dataflows besides the [!DNL SugarCRM Accounts & Contacts] card menu on the Catalog page. You can then select [!UIControl Preview dataset] to verify the data that was ingested.
+
+1. Depending on the object type you are working with you can verify the aggregated data against the counts visible on the [!DNL SugarMarket] Accounts page below : 
+   ![Screenshot from the SugarMarket Accounts page displaying list of accounts](../../../../images/tutorials/create/sugarcrm-accounts-contacts/sugarmarket-accounts.png?lang=en)
+
+   Or from the Contacts page below :
+   ![Screenshot from the SugarMarket Contacts page displaying list of contacts](../../../../images/tutorials/create/sugarcrm-accounts-contacts/sugarmarket-contacts.png?lang=en)
+
+>[!NOTE] 
+> The [!DNL SugarMarket] pages do not include the deleted object counts. However, data retrieved through this source will also include the deleted count, these would be marked with a deleted flag.
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/crm/sugarcrm-accounts-contacts.md]: Need to add another ">" in between here, and also need to remove the space between ">" and the text, so:

>[!NOTE]
>
>The [!DNL SugarMarket]...

File: help/sources/tutorials/ui/create/crm/sugarcrm-accounts-contacts.md [hunk 11]
```diff
@@ -0,0 +1,109 @@
...
+
+### Guardrails {#guardrails}
+
+The [!DNL SugarCRM] API throttle rates are 90 calls per minute or 2000 calls per day, whichever happens first. However, this restriction has been circumvented by adding a parameter into the connection spec that will delay request time so that the rate limit is never reached.
+
+### Validation {#validation}
+
+To validate that you have correctly set up the source and [!DNL SugarCRM Accounts & Contacts] data is being ingested, follow the steps below:
+
+1. In the Platform UI, select View Dataflows besides the [!DNL SugarCRM Accounts & Contacts] card menu on the Catalog page. You can then select [!UIControl Preview dataset] to verify the data that was ingested.
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/crm/sugarcrm-accounts-contacts.md]: Need **[!UICONTROL]** around View Dataflows

File: help/sources/tutorials/ui/create/crm/sugarcrm-events.md [hunk 12]
```diff
@@ -0,0 +1,91 @@
...
+
+### Guardrails {#guardrails}
+
+The [!DNL SugarCRM] API throttle rates are 90 calls per minute or 2000 calls per day, whichever happens first. However, this restriction has been circumvented by adding a parameter into the connection spec that will delay request time so that the rate limit is never reached.
+
+### Validation {#validation}
+
+To validate that you have correctly set up the source and [!DNL SugarCRM Events] data is being ingested, follow the steps below:
+
+1. In the Platform UI, select View Dataflows besides the [!DNL SugarCRM Events] card menu on the Catalog page. You can then select [!UIControl Preview dataset] to verify the data that was ingested.
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/crm/sugarcrm-events.md:70]: Need **[!UICONTROL]** around View Dataflows

**Review by carlojavier — CHANGES_REQUESTED** [review]
Hi @mohan-h review completed. Looking good so far! Pls let me know if you have any questions

File: help/sources/tutorials/api/create/crm/sugarcrm-accounts-contacts.md (hunk 2 above)
Comment (mohan-h) [reply help/sources/tutorials/api/create/crm/sugarcrm-accounts-contacts.md]: * It appears the account can only be obtained through an Account manager. Have mentioned the same.
* Thanks the rewording, reads better.

File: help/sources/tutorials/ui/create/crm/sugarcrm-accounts-contacts.md (hunk 6 above)
Comment (mohan-h) [reply help/sources/tutorials/ui/create/crm/sugarcrm-accounts-contacts.md]: Removed from UI
  ```help/sources/tutorials/ui/create/crm/sugarcrm-accounts-contacts.md```
Have only preserved it within the API
  ```help/sources/tutorials/api/create/crm/sugarcrm-accounts-contacts.md```

File: help/sources/tutorials/api/create/crm/sugarcrm-events.md (hunk 7 above)
Comment (mohan-h) [reply help/sources/tutorials/api/create/crm/sugarcrm-events.md]: Removed from the API
  ```help/sources/tutorials/api/create/crm/sugarcrm-events.md```

File: help/sources/tutorials/ui/create/crm/sugarcrm-events.md (hunk 8 above)
Comment (mohan-h) [reply help/sources/tutorials/ui/create/crm/sugarcrm-events.md]: Removed from UI
  ```help/sources/tutorials/ui/create/crm/sugarcrm-events.md```

File: help/sources/tutorials/api/create/crm/sugarcrm-accounts-contacts.md (hunk 5 above)
Comment (mohan-h) [reply help/sources/tutorials/api/create/crm/sugarcrm-accounts-contacts.md:586]: Introduced on both the UI and API pages for the SugarCRM Accounts & Contacts connector.
